pyproject.toml

```diff
@@ -59,6 +59,10 @@ libcst = "^0.4.9"
 
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
+log_level = "ERROR"
+log_cli = true
+log_cli_format = "%(asctime)s [%(levelname)s] %(message)s"
+log_cli_date_format = "%Y-%m-%d %H:%M:%S.%f"
```

Comment:
what is the scope of these settings? Is this for like test runs, or does this get included in the SDK and control logging config for users?

Reply:
This is just for the test runs

Reply:
To verify you can look at line 60 `tool.pytest.ini_options`

Reply:
🤦 duh thank you

Reply:
Very cool!